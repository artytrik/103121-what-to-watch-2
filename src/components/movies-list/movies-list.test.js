import React from 'react';
import renderer from 'react-test-renderer';
import MoviesList from './movies-list.jsx';

it(`MoviesList correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<MoviesList
      movies = {[
        {
          id: 1,
          name: `Fantastic Beasts: The Crimes of Grindelwald`,
          genre: `Comedy`,
          preview: `/img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
          link: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
        }
      ]}
    />, {
      createNodeMock: () => {
        return {};
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
