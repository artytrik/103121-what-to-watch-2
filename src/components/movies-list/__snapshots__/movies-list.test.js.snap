// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesList correctly renders after relaunch 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt=""
        height="175"
        src=""
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="#"
        onClick={[Function]}
      >
        
      </a>
    </h3>
  </article>
</div>
`;
